app/src/main/java/com/nevilleantony/prototype/storage/DownloadsDatabase.java

```diff
@@ -6,23 +6,25 @@
 import androidx.room.Room;
 import androidx.room.RoomDatabase;
 
-@Database(entities = {DownloadsModel.class}, version = 1)
-public abstract class DownloadsDatabase extends RoomDatabase{
-    public abstract DownloadsDao getDoa();
+
+@Database(entities = {DownloadsModel.class}, version = 1, exportSchema = false)
+public abstract class DownloadsDatabase extends RoomDatabase {
     public static DownloadsDatabase dwld_db;
 
-    public static DownloadsDatabase getInstance(Context context){
-        if(dwld_db == null){
-            dwld_db = buildDatabaseInstance(context);
-        }
+    public static DownloadsDatabase getInstance(Context context) {
+//        if(dwld_db == null){
+        dwld_db = buildDatabaseInstance(context);
+//        }
         return dwld_db;
     }
 
-    public static DownloadsDatabase buildDatabaseInstance(Context context){
-        return Room.databaseBuilder(context, DownloadsDatabase.class, "Sample.db").allowMainThreadQueries().build();
+    public static DownloadsDatabase buildDatabaseInstance(Context context) {
+        return Room.databaseBuilder(context, DownloadsDatabase.class, "Sample").allowMainThreadQueries().build();
```

Comment:
Why not move this into constructor as it should not be called from anywhere else